src/collection-view.js

```diff
@@ -482,7 +487,13 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // render the child view
   renderChildView: function(view, index) {
+    if (!(view instanceof Marionette.View)) {
+      Marionette.triggerMethodOn(view, 'before:render', view);
+    }
     view.render();
+    if (!(view instanceof Marionette.View)) {
```

Comment:
Also is `view instanceof Mn.View` the best way to do this or should we check for the presence of individual methods/hooks